core/src/block_creation_loop.rs

```diff
@@ -377,14 +379,16 @@ fn record_and_complete_block(
 
     // Construct and send the block footer
     let mut w_poh_recorder = poh_recorder.write().unwrap();
+    let bank = w_poh_recorder
+        .bank()
+        .expect("Bank cannot have been cleared as BlockCreationLoop is the only modifier");
+
+    let bank_hash = bank.hash();
```

Comment:
nit: can you add comments here why we definitely have bank frozen here?

Reply:
we don't.
Since we're no longer doing parent bank hash but the current bank hash, this becomes tricky.
We'll have to produce the footer in broadcast when detecting the alpentick.

Reply:
another alternative is to have `tick_alpenglow` hold on sending the last tick until replay has frozen the bank. At that point we send the footer + last tick to broadcast.

Reply:
shouldn't be too bad, need to wait here for the bank to be frozen, only if we're flushing cache due to alpentick
https://github.com/anza-xyz/alpenglow/blob/c3d497595c41079710b3ca36601f33a1cc142afb/poh/src/poh_recorder.rs#L603-L609

then along with sending the alpentick we send the footer